EDA_analysis.Rmd

```diff
@@ -0,0 +1,80 @@
+
+## EDA Table
+
+Each row in the data set represents COVID-19 related information for a given country on a given day starting from 2019-12-31. To limit the scope of our project, we have chosen the data set that includes information up to 2020-11-18. Within this data set, there are 57,606 observations and 50 variables. We will be primarily looking at the COVID-19 data on Canada and USA. Since we are interested in both number of new cases and number of new tests, any row with missing values in either column is removed from the date set. We also calculated the response ratio as defined below:
+
+Response Ratio = \# of daily new tests / \# of daily new cases
+
+Below we have shown the number of observations, mean, median and standard deviation of response ratio (Table 1).
+
+The mean and median ratio for the USA are close to each other, which suggests that the ratio for the USA is normally distributed. The mean response ratio for Canada is higher than the median, so its distribution seems to be right skewed. This suggests that the true population of response ratio may not be normally distributed. However, since the sample size is well above 30, the large enough sample size assumption under the Central Limit Theorem is met. Therefore the sample observation does not need to come from a normally distributed population. Because all conditions of Central Limit Theorem have been met, we decided that it is safe to use mean response ratio as test statistic for our hypothesis test.
+
+The standard deviation of Canada is quite large, compared with the mean and median ratio. It indicates that the response ratio in Canada is more widely spread out. The standard deviation of the USA is a lot smaller, which suggests that more of the ratio is clustered around the mean. It should be noted that the standard deviation might have been affected by data in the early stage of the pandemic, as tests were not fully developed and new cases were only detected within a few communities.
+
+```{r EDA Table}
+covid_all_country <-  read_csv("data/owid-covid-data.csv", guess_max = 5000)
+
+covid_CAN_USA <- covid_all_country %>% 
+    filter(iso_code=="CAN"| iso_code =="USA") %>% 
+    filter(date < "2020-11-01") %>% 
```

Comment:
I think we also need to filter for `date > "2020-03-11"`for Canada since we decided that the upper bound for Canada will be the first non-NaN value in new cases.